examples/Phonon_Caustics/src/Caustic_PhononDetectorConstruction.cc

```diff
@@ -145,8 +145,8 @@ G4VSolid* SubstrateSolid= new G4Box("SubstrateSolid",0.2*cm,0.2*cm,0.2*cm);
 
 G4LogicalVolume* BolometerLogical =
```

Comment:
You don't need to change it, but why exactly did you remove the leading "f" from the name of the variable?  That is a standard convention throughout the Geant4 software, and I was not suggesting that you change it.  I was _only_ suggesting that you change the *string* name assigned to the various volumes.  Those *strings* (not the local C++ variable names) can be printed out using standard Geant4 macro commands:
```
/vis/open ATree
/vis/ASCIITree/verbose 12
/vis/drawTree worlds
```
so the *strings* are useful as human readable documentation for the geometry that is being built.

Don't bother changing things back.  This has already dragged on too long.  I'm sorry that I wasn't completely clear about what I was asking for.

Reply:
I changed the variable's name because it is not a global variable defined in the Class.